lessons/specifics/guardian.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: page
+title: Guardian (Basics)
+category: specifics
```

Comment:
I reckon we need to move this out into an external section, maybe something like **External Libraries**, or **Hex Packages**